components/channel_notifications_modal/channel_notifications_modal.jsx

```diff
@@ -50,7 +59,7 @@ export default class ChannelNotificationsModal extends React.Component {
 
     handleSubmitDesktopNotifyLevel() {
```

Comment:
Please change this to arrow function `handleSubmitDesktopNotifyLevel = () => {...}` and then remove function bindings at `constructor`. Do the same on other functions.